platform/linux-generic/odp_impl.c

```diff
@@ -15,7 +15,7 @@
 	ODP_VERSION_TO_STR(ODP_VERSION_API_GENERATION) "." \
 	ODP_VERSION_TO_STR(ODP_VERSION_API_MAJOR) "." \
 	ODP_VERSION_TO_STR(ODP_VERSION_API_MINOR) ") " \
-	ODP_VERSION_TO_STR(ODP_VERSION_BUILD)
+	ODP_VERSION_TO_STR(ODP_VERSION_BUILD) _ODP_VERSION_GIT_REVISION \
```

Comment:
The `odp_version_impl_str()` output includes quite much duplicate information:
E.g. `odp-linux 1.46.0-0 (v1.46.0) 1.46.0.0-87d04d69`

This could be condensed to:
`odp-linux-1.46.0.0-87d04d69`